### PR TITLE
Use ES module imports instead of HTML import

### PR DESCRIPTION
**Pull request by feynmanliang**
This change is [Reviewable](https://reviewable.io/reviews/polymerelements/paper-icon-button/147)

### CODE REVIEW

**Comment by bicknellr** [comment]
I'm changing the base branch to `master` so that we can delete `__auto_generated_3.0_preview`.

**Comment by feynmanliang** [comment]
Ok thanks! FWIW I had an earlier PR #144 against `master` closed.

File: README.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ Material design: [Icon toggles](https://www.google.com/design/spec/components/bu
 the button, a ripple effect emanates from the center of the button.
 
 `paper-icon-button` does not include a default icon set. To use icons from the default
-set, include `PolymerElements/iron-icons/iron-icons.html`, and use the `icon` attribute to specify which icon
+set, include `PolymerElements/iron-icons/iron-icons.js`, and use the `icon` attribute to specify which icon
```
Comment (bicknellr) [inline README.md]: `PolymerElements` -> `@polymer`:
https://github.com/PolymerElements/paper-icon-button/blob/39f10482a379e9f98c9238cb4c907416868440ef/demo/index.html#L18

Comment (feynmanliang) [reply README.md]: Changed

**Pull request merged**